lesson24/script.js

```diff
@@ -0,0 +1,199 @@
+//1.Подсчитать сумму всех чисел в заданном пользователем диапазоне.
+
+let firstNumber = prompt("Введите пожалуйста первое число:");
+let secondNumber = prompt("Введите пожалуйста Второе число:");
+let sum = 0;
+for (i = firstNumber; i <= secondNumber; i++) {
+  sum += firstNumber++;
+}
+alert(`Ваш ответ: ${sum}`);
+
+//2.Запросить 2 числа и найти только наибольший общий делитель.
+
+let firstNumber = prompt("Введите пожалуйста первое число:");
+let secondNumber = prompt("Введите пожалуйста Второе число:");

```

Comment:
привести к числу, чтоб не было неожиданностей
```suggestion
let firstNumber = +prompt("Введите пожалуйста первое число:");
let secondNumber = +prompt("Введите пожалуйста Второе число:");
```